services/ingestion/etl/budg/csv/src/lib/transform.js

```diff
@@ -1,11 +1,20 @@
 /*
- * Transform message (BUDGCSV)
+ * Transform message (BUDG CSV)
  */
 
 /*
  * Map fields
  */
 export default record => {
+  // Preprocess budget
+  const budgetObject = {
+    total_cost: null,
+    eu_contrib: Number(record['EU Budget contribution']),
```

Comment:
In this specific etl, it seems to be always a string (in fact it can't really be something else, coming from a csv file).
We could add some tests (for each field in fact), but I assumed that the data model will be coherent.
We also intended to add some data model verification at a later stage, so maybe we could do this at this time.